### PR TITLE
osdocs-414 machine limitations on user-provisioned infra

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://jira.coreos.com/browse/OSDOCS-414

@enxebre, will you PTAL? 

The built draft is here: http://file.rdu.redhat.com/kalexand/061119/osdocs414/machine_management/creating-machineset.html

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@enxebre, will you PTAL?

**Comment by kalexand-rh** [comment]
@enxebre, will you PTAL?

File: modules/machine-user-provisioned-limitations.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//
+// * machine_management/applying_autoscaling.adoc
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating-machinesets.adoc
+// * machine_management/deploying-machine-health-checks.adoc
+// * machine_management/manually-scaling-machinesets.adoc
+
+[IMPORTANT]
+====
+This process is not applicable to clusters that use user-provisioned
```
Comment (enxebre) [inline modules/machine-user-provisioned-limitations.adoc]: nothing would stop you from running machine scalable resources in e.g upi aws if you configure it properly

Comment (kalexand-rh) [reply modules/machine-user-provisioned-limitations.adoc]: @enxebre, that's been a contentious point during reviews before GA. Do you think that it's time to document that path?

**Comment by kalexand-rh** [comment]
@sunzhaohua2, will you PTAL?

**Comment by sunzhaohua2** [comment]
@kalexand-rh sorry, I missed this pr, I agree with enxebre. This process is not applicable to  bare metal or vSphere UPI, upi aws if you configure it properly it's ok.

**Comment by kalexand-rh** [comment]
I've made an issue for us to document configuring machine scalable resources in AWS and GCP: https://jira.coreos.com/browse/OSDOCS-708

I'm going to merge this PR for now and follow up on that Jira.

**Pull request merged**